packages/portal/src/components/feedback/FeedbackWidget.vue

```diff
@@ -380,7 +381,7 @@
       position: fixed;
       right: 1rem;
       bottom: 1rem;
-      z-index: 1000;
+      z-index: 10000;
```

Comment:
this seems excessive. For what scenario was this needed? I think we should be able to move whatever other element was in front of this to have a lower z-index instead.

Reply:
Indeed. It was unintentional. Reverted.